gson/src/main/java/com/google/gson/JsonParser.java

```diff
@@ -26,7 +26,47 @@
 
 /**
  * A parser to parse JSON into a parse tree of {@link JsonElement}s.
+ * 
```

Comment:
Maybe it would also be useful to mention here that the JSON data is parsed in lenient mode (though each of the methods already says this as well).
```suggestion
 * 
 * <p>The JSON data is parsed in {@linkplain JsonReader#setStrictness(Strictness) lenient mode}.
 *
```